### ISSUE TITLE
snapshot creation fails with `Size is not a multiple of 512` error

### ISSUE THREAD

**Issue by aleofreddi** [aleofreddi/csi-sanlock-lvm#34]
Using `maxSizePct` can lead to snapshot sizes that are not multiple of 512b, causing the snapshot to fail with the following error:

```
Message:     Failed to check and update snapshot content: failed to take snapshot of the volume csl-v-Ep1UaCzhbQEP1ovLaYemh1bZzPXF68JIG+nqbdBJaKg@vg_ssd: "rpc error: code = Unknown desc = rpc error: code = Unknown desc = unexpected error: rc=3 stdout=\"\" stderr=\"  Size is not a multiple of 512. Try using 429496320 or 429496832.\\n  Invalid argument for --size: 429496729b\\n  Error during parsing of command line.\\n\""
```

**Comment by aleofreddi** [comment]
Fixed in 0950c9546783654c4327cc0e3910ee05f9419abf.

**Comment by aleofreddi** [comment]
Fixed in b268542c0539a137b362a4c8d22834c30bd58ab6.
